Assets/HoloToolkit/Input/Scripts/Gaze/GazeManager.cs

```diff
@@ -12,6 +12,32 @@ namespace HoloToolkit.Unity.InputModule
     /// </summary>
     public class GazeManager : Singleton<GazeManager>, IPointingSource
     {
+        [Obsolete("Use FocusManager.PointerSpecificFocusChangedMethod")]
+        public delegate void FocusedChangedDelegate(GameObject previousObject, GameObject newObject);
+
+        /// <summary>
+        /// Indicates whether the user is currently gazing at an object.
+        /// </summary>
+        [Obsolete("Use FocusManager.TryGetFocusDetails")]
+        public bool IsGazingAtObject { get; private set; }
+
+        /// <summary>
+        /// Dispatched when focus shifts to a new object, or focus on current object
+        /// is lost.
+        /// </summary>
+        [Obsolete("Use FocusManager.PointerSpecificFocusChanged")]
+#pragma warning disable 618
```

Comment:
Doesn't this disable the above Obsolete warning?

Reply:
No, it's wrapped around just the one field and the other obsolete warnings still show.

We disable then re-enable a few lined after.

Reply:
Right, but why is this event marked both Obsolete and has the Obsolete warning disabled?

Reply:
It's to disable the Obsolete warning from using the `FocusedChangedDelegate`, but it doesn't disable the warning when subscribing to the event.

Reply:
Ahh sweet!